package.json

```diff
@@ -14,48 +14,54 @@
     "eslint": "eslint --fix \"src/**/*.{ts,tsx}\"",
     "prettier": "prettier --write \"src/**/*.{js,jsx,ts,tsx}\"",
     "test": "vitest run",
-    "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook",
+    "ladle": "ladle serve -p 6006",
+    "build-ladle": "ladle build -o dist-ladle",
+    "tsc": "tsc --noEmit",
+    "tsc -w": "tsc --noEmit --watch",
```

Comment:
Unlike Yarn, you cannot run scripts directly from your `node_modules` using npm, this means that commands that work in yarn like `yarn tsc` will not work if you run `npm run tsc`. To get around this we create a script called `tsc` which then references `tsc`